### PR TITLE
fix(ml): handle missing `context_length` field

### PR DESCRIPTION
**Pull request by mertalev**
## Description

A few models seem not to have `context_length` in their `config.json` file, so search will fail for these models. This PR sets a default of 77 in these cases.

## How Has This Been Tested?

Tested by running Smart Search and searching with `nllb-clip-large-siglip__v1` and `XLM-Roberta-Large-ViT-H-14__frozen_laion5b_s13b_b90k`.

### CODE REVIEW

File: machine-learning/app/config.py [hunk 1]
```diff
@@ -81,6 +83,7 @@ def __init__(self) -> None:
             console=console,
             rich_tracebacks=True,
             tracebacks_suppress=[*self.excluded, concurrent.futures],
+            tracebacks_show_locals=LOG_LEVEL == logging.DEBUG,
```
Comment (jrasm91) [inline machine-learning/app/config.py:86]: Is this change intentional or only for debugging?

**Review by jrasm91 — APPROVED** [review]

File: machine-learning/app/config.py (hunk 1 above)
Comment (mertalev) [reply machine-learning/app/config.py:86]: Intentional since it makes debugging this sort of thing easier
